cmd/calico-node/main.go

```diff
@@ -65,6 +66,10 @@ var felixReady = flagSet.Bool("felix-ready", false, "Run felix readiness checks"
 // thresholdTime is introduced for bird readiness check. Default value is 30 sec.
 var thresholdTime = flagSet.Duration("threshold-time", 30*time.Second, "Threshold time for bird readiness")
 
+// Options for node status.
+var runStatusReporter = flagSet.Bool("status-reporter", false, "Run node status reporter")
+var showStatus = flagSet.Bool("show-status", false, "Print out node status")
```

Comment:
`show-status` would allow somebody to quickly checking node status by 
```
kubectl exec -t calico-node-xxx -n calico-system -- calico-node -show-status
```